doc/sphinx/language/gallina-extensions.rst

```diff
@@ -2443,11 +2443,15 @@ The reduction machines (:tacn:`vm_compute`, :tacn:`native_compute`) implement
 dedicated, efficient, rules to reduce the applications of these primitive
 operations.
 
-These primitives, when extracted to OCaml (see :ref:`extraction`), are mapped to
-types and functions of a :g:`Uint63` module. Said module is not produced by
-extraction. Instead, it has to be provided by the user (if they want to compile
-or execute the extracted code). For instance, an implementation of this module
-can be taken from the kernel of Coq.
+The extraction of these primitives can be customized similarly to the extraction
```

Comment:
Is it explained somewhere how constants are extracted? eg `Definition foo : Int63.int := 12. Extraction foo.`

Reply:
Done. Thanks.